org.freedesktop.LinuxAudio.Plugins.NeuralAmpModeler.yml

```diff
@@ -0,0 +1,30 @@
+id: org.freedesktop.LinuxAudio.Plugins.NeuralAmpModeler
+branch: '22.08'
+build-extension: true
+runtime: org.freedesktop.LinuxAudio.BaseExtension
+runtime-version: stable
+sdk: org.freedesktop.Sdk//22.08
+build-options:
+  prefix: /app/extensions/Plugins/NeuralAmpModeler
+modules:
+  - name: NeuralAmpModeler
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    post-install:
+      - "mv ${FLATPAK_DEST}/lib/lv2 ${FLATPAK_DEST}"
+      - "install -Dm644 LICENCE.md ${FLATPAK_DEST}/share/licenses/NeuralAmpModeler/LICENSE"
+      - "install -Dm644 deps/NeuralAmpModelerCore/LICENSE ${FLATPAK_DEST}/share/licenses/NeuralAmpModeler/LICENSE"
```

Comment:
the second line clobber the first one.

Reply:
Good catch, thanks! I'll move the first license to the `NeuralAmpModelerLV2Plugin` subdirectory instead.